src/utils/d2c_messaging/inc/aduc/d2c_messaging.h

```diff
@@ -23,43 +23,47 @@ EXTERN_C_BEGIN
 typedef enum _tagADUC_D2C_Message_Type
 {
     ADUC_D2C_Message_Type_Device_Update_Result = 0, /**< deviceUpdate interface reported property */
-    ADUC_D2C_Message_Type_Device_Update_ACK,        /**< deviceUpdate interface ACK */
-    ADUC_D2C_Message_Type_Device_Information,       /**< deviceInformation interface reported property */
-    ADUC_D2C_Message_Type_Diagnostics,              /**< diagnostics interface reported property */
-    ADUC_D2C_Message_Type_Diagnostics_ACK,          /**< diagnostics interface ACK */
-    ADUC_D2C_Message_Type_Device_Properties,        /**< deviceUpdate interface reported property */
+    ADUC_D2C_Message_Type_Device_Update_ACK, /**< deviceUpdate interface ACK */
+    ADUC_D2C_Message_Type_Device_Information, /**< deviceInformation interface reported property */
+    ADUC_D2C_Message_Type_Diagnostics, /**< diagnostics interface reported property */
+    ADUC_D2C_Message_Type_Diagnostics_ACK, /**< diagnostics interface ACK */
+    ADUC_D2C_Message_Type_Device_Properties, /**< deviceUpdate interface reported property */
     ADUC_D2C_Message_Type_Max
 } ADUC_D2C_Message_Type;
 
 typedef enum _tagADUC_D2C_Message_Status
 {
-    ADUC_D2C_Message_Status_Pending = 0,            /**< Waiting to be processed */
-    ADUC_D2C_Message_Status_In_Progress,            /**< Being processed by the messages processor */
-    ADUC_D2C_Message_Status_Waiting_For_Response,   /**< Sent to the cloud, waiting for response */
-    ADUC_D2C_Message_Status_Success,                /**< Message has been processed successfully */
-    ADUC_D2C_Message_Status_Failed,                 /**< A failure occurred. No longer process */
-    ADUC_D2C_Message_Status_Replaced,               /**< Message was replaced by a new one */
-    ADUC_D2C_Message_Status_Canceled,               /**< Message was canceled */
-    ADUC_D2C_Message_Status_Max_Retries_Reached,    /**< Maximum number of retries reached */
+    ADUC_D2C_Message_Status_Pending = 0, /**< Waiting to be processed */
+    ADUC_D2C_Message_Status_In_Progress, /**< Being processed by the messages processor */
```

Comment:
nit: Explicit settings can make it easier to debug some enums (eg the connection reasons) if these are ever made into umock enums for unit testing